managers/tests/test_unit_managers_apiview.py

```diff
@@ -37,6 +37,7 @@ def setUp(self):
         self.url = [
             reverse('custom-projects-list'),
             reverse('custom-project-detail', args=(self.project.pk,)),
+            reverse('custom-project-update', args=(self.project.pk,)),
```

Comment:
Same here, please overwrite `self.url` in `ProjectUpdateTests` `setUp` method.

Reply:
Done